helm/dagster/templates/configmap-instance.yaml

```
@@ -105,5 +105,16 @@ data:
       {{- end }}
     {{- end }}
 
+    {{- if and (.Values.dagsterDaemon.enabled) (.Values.dagsterDaemon.sensors) }}
+    {{- $sensors := .Values.dagsterDaemon.sensors }}
+    {{- if $sensors.useThreads }}
+    sensors:
+      use_threads: {{ $sensors.useThreads }}
+      {{- if $sensors.numWorkers }}
+      num_workers: {{ $sensors.numWorkers }}
+      {{- end }}
+    {{- end }}
+    {{- end }}
```

We can simplify this because we know `.Values.dagsterDaemon.sensors` will always be defined, because of our schema.

```suggestion
    {{- $sensors := .Values.dagsterDaemon.sensors }}
    {{- if and (.Values.dagsterDaemon.enabled) ($sensors.useThreads) }}
    sensors:
      use_threads: {{ $sensors.useThreads }}
      {{- if $sensors.numWorkers }}
      num_workers: {{ $sensors.numWorkers }}
      {{- end }}
    {{- end }}
```